lib/extensions/ar_migration.rb

```diff
@@ -1,7 +1,6 @@
 module ArPglogicalMigrationHelper
   def self.discover_schema_migrations_ran_class
-    raise "Remove 'silence' block now that Rails version is >= 5.1" if Rails.version >= "5.1"
-    ActiveSupport::Deprecation.silence { return unless ActiveRecord::Base.connection.table_exists?("schema_migrations_ran") }
+    return unless ActiveRecord::Base.connection.table_exists?("schema_migrations_ran")
```

Comment:
This revert will be removed before merge... it allowed us to see these deprecations in travis